doc/_extensions/autojsdoc/cgitb.py

```diff
@@ -0,0 +1,319 @@
+"""More comprehensive traceback formatting for Python scripts.
```

Comment:
As said in main comment, duplicating `cgitb` is vastly overkill.

Reply:
Dropped in #36463